server/controllers/article.controllers.js

```diff
@@ -255,13 +255,101 @@ const createHighlight = async (req, res) => {
   });
 };
 
+const reviewArticle = async (req, res) => {
+  const reviewedArticle = await ReportedArticle.findOne({
+    where: {
+      reported_article_id: req.params.articleId,
+    },
+  });
+
+  if (!reviewedArticle) {
+    return res.status(404).json({
+      errors: {
+        body: ['Article not found'],
+      },
+    });
+  }
+
+  if (reviewedArticle.status === 'reviewed') {
+    return res.status(409).json({
+      errors: {
+        body: ['This article has already been reviewed'],
+      },
+    });
+  }
+
+  const values = {
+    reviewer_comment: req.body.reviewer_comment,
+    status: 'reviewed',
+  };
+
+  const updatedReview = await reviewedArticle.update(values);
+
+  return res.status(200).json({
+    updatedReview,
+  });
+};
+
+const articleStatus = async (req, res) => {
+  try {
+    const { articleId } = req.params;
+    const reviewedArticle = await ReportedArticle.findOne({
+      where: { reported_article_id: articleId },
```

Comment:
Add a where statement to check for the article id and also where the status is `reviewed`